overlord/booted.go

```diff
@@ -17,12 +17,14 @@
  *
  */
 
-package snappy
+package overlord
```

Comment:
I'm a bit concerned to move this into the general overlord package. It may easily become the "snappy package v2" if we're not clear on what should land there.

How about creating an `overlord/boot` package holding with `InitialSetup` (old `Firstboot`) and `UpdateRevisions` (old `Sync`) functions?